doc/moduleAT.dox

```diff
@@ -0,0 +1,457 @@
+    + \frac{\lambda}{4\varepsilon} \langle 1 - v , 1 - v \rangle_0.
+\f]
+
+
+
+A second possibility is to define \f$ u \f$ and \f$ g \f$ on the vertices
+and \f$ v \f$ on the edges. We denote this formulation
+AT10. Gray levels are seen as point mass on the center of pixels, so
+that functions \f$ u \f$ and \f$ g \f$ are both 0-forms, while \f$ v \f$
+is a dual 1-form in between \f$ u \f$. It follows:
+\f[
+  \displaystyle
+  AT10(u,v) = \Sigma_{i=1}^n
+      \alpha \langle u_i - g_i , u_i - g_i \rangle_0
+    + \langle v , \mathbf{d_0} u_i \rangle_1 \langle v ,
+                                          \mathbf{d_0} u_i \rangle_1 \\
+    + \lambda \varepsilon \langle (\mathbf{d_1}
+          + \bar{\mathbf{\star}} \bar{\mathbf{d_1}} \mathbf{\star}) v ,
+          (\mathbf{d_1} + \bar{\mathbf{\star}} \bar{\mathbf{d_1}}
+                        \mathbf{\star}) v   \rangle_1
+    + \frac{\lambda}{4\varepsilon} \langle 1 - v , 1 - v \rangle_1.
+\f]
+
+For more details about discrete formulations and numerical
+optimizationf of AT, we refer the reader to [2] and [3].
+
+@subsection AT2_3 Solutions to AT01 and AT20
+
+Functionals AT20 and AT01 are both sums of quadratic terms, independently,
+but not simultaneously, convex in \f$ u \f$ and \f$ v \f$. They must have
+null derivative at optimum. We thus propose to alternatively solve for
+\f$ u \f$, then \f$ v \f$. The derivatives can be given explicitly as
+linear systems.
+To simplify notations, let \f$ \mathbf{A} := \mathbf{d_0} \f$,
+\f$  \mathbf{B} := \mathbf{d_1} \f$,
+\f$ \mathbf{A'} := \bar{\mathbf{\star}} \bar{\mathbf{d_1}} \mathbf{\star} \f$
+and \f$ \mathbf{B'} := \bar{\mathbf{\star}} \bar{\mathbf{d_0}} \mathbf{\star} \f$.
+We get at optimum:
+\f[
+  \displaystyle
+  \left\{ \begin{array}{r @{=} l c}
+    \left[\alpha \mathbf{Id} - {\mathbf{B'}}^T diag(\mathbf{M} v)^2 \mathbf{B'} \right] u
+    &\alpha g & (1.1) \\
+    \left[ \frac{\lambda}{4\varepsilon} \mathbf{Id}
+          + \lambda \varepsilon \mathbf{A}^T \mathbf{A}
+          + \mathbf{M}^T diag(\mathbf{B'} u)^2 \mathbf{M} \right] v
+    &\frac{\lambda}{4 \varepsilon} \mathbf{1}
+    & (1.2)
+  \end{array}\right.
+\f]
+
+for the derivative of AT20, and
+
+\f[
+  \displaystyle
+  \left\{ \begin{array}{r @{=} l c}
+    \left[\alpha \mathbf{Id} - \mathbf{A}^T diag(v)^2 \mathbf{A} \right] u
+    &\alpha g & (2.1) \\
+    \left[ \frac{\lambda}{4\varepsilon} \mathbf{Id}
+          + \lambda \varepsilon (\mathbf{A'}^T \mathbf{A'}
+          + \mathbf{B}^T \mathbf{B})
+          + diag(\mathbf{A} u)^2  \right] v
+    &\frac{\lambda}{4 \varepsilon} \mathbf{1}
+    & (2.2)
+  \end{array}\right.
+\f]
+for the derivative of AT10. Since all matrices are
+symmetric, definite and positive, we use a Cholesky factorization to
+solve alternatively the two equations of each system.
+
+
+@subsection AT2_4 Algorithm
+
+
+\f$
+OPTIMIZE-AT01( (g_1, ..., g_N ): 0-forms,
+(\alpha, \lambda, \varepsilon_1, \varepsilon_2, \varepsilon_r): reals );
+\f$
+
```

Comment:
Here after it proceed that on my doc: (with doxyen 1.8.11)
![capture d ecran 2016-11-05 a 15 10 42](https://cloud.githubusercontent.com/assets/772865/20030676/30ceeec6-a36a-11e6-9d05-2de586926bad.png)